cw-orch/Cargo.toml

```diff
@@ -66,6 +66,11 @@ osmosis-test-tube = ["dep:osmosis-test-tube"]
 # Default deps
 cw-orch-contract-derive = { path = "../packages/cw-orch-contract-derive", version = "0.13.3" }
 cw-orch-fns-derive = { path = "../packages/cw-orch-fns-derive", version = "0.13.3" }
+interactive-clap = "0.2.4"
```

Comment:
if these are only used in the CLI package then we should not put them in the workspace toml.